demisto_sdk/yaml_tools/content_creator.py

```diff
@@ -0,0 +1,321 @@
+import os
+import glob
+import shutil
+import zipfile
+import io
+import yaml
+from ..common.constants import INTEGRATIONS_DIR, MISC_DIR, PLAYBOOKS_DIR, REPORTS_DIR, DASHBOARDS_DIR, \
+    WIDGETS_DIR, SCRIPTS_DIR, INCIDENT_FIELDS_DIR, CLASSIFIERS_DIR, LAYOUTS_DIR, CONNECTIONS_DIR, \
+    BETA_INTEGRATIONS_DIR, INDICATOR_FIELDS_DIR, INCIDENT_TYPES_DIR, TEST_PLAYBOOKS_DIR, PACKS_DIR, DIR_TO_PREFIX
+from ..common.tools import get_child_directories, get_child_files, print_warning
+from .unifier import Unifier
+
+
+class ContentCreator:
+
+    def __init__(self, artifacts_path: str, content_bundle_path='',
+                 test_bundle_path='', packs_bundle_path='', preserve_bundles=False):
```

Comment:
do we need all these arguments as we're not sending them from the core?

Reply:
no, but it does allow you if you wanted to import `ContentCreator` class for some reason in your code somewhere and wanted to set the paths for the bundles then you could do that